nmigen/vendor/xilinx_spartan_3_6.py

```diff
@@ -76,6 +81,10 @@ class XilinxSpartan6Platform(TemplatedPlatform):
             -ifn {{name}}.prj
             -ofn {{name}}.ngc
             -top {{name}}
+            {% if platform.family in ["3", "3E", "3A"] %}
+            -use_new_parser yes
```

Comment:
What does this switch does for Spartan 6? Should we not always use it? Is that a part of ISE or just its Spartan support?

Reply:
I was under the impression that you don't add any options you don't have to the templates (e.g. that are default for the family). E.g. while `-use_new_parser yes` is the default for S6 already (and thus harmless to put on the command line), it is _not_ the default for S3.

Reply:
OK, that's fine then.